concepts/05 Widgets/Form/01 Getting Started with Form/80 Submit the Form.md

```diff
@@ -0,0 +1,325 @@
+To submit the Form, wrap it in the HTML <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form" target="_blank">form</a> element. Then add the [button item](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/ButtonItem/) and set its **buttonOptions**.[useSubmitBehavior](/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#useSubmitBehavior) property to **true**. If you specify any validation rules, you can submit the Form data to the server only when all validation checks pass successfully.  
```

Comment:
We shouldn't describe code with so many words, only mention the important points. Look at how the same information is conveyed in the old topic:

> Usually, Form editors should be submitted to the server after being successfully validated on the client. The following code shows how to do this using a button form item. Note that the Form widget is wrapped in the `<form>` tag in the markup.